### PR TITLE
BSD fixes #84: Case studies story

### LINKED ISSUE #84: Refine Case Study page
Related to #53. Need to refine Case Study page once design is finalized.

### PR DESCRIPTION
**Pull request by mejiaj**
This PR updates the Case Studies story with new design. It also includes a minor update to the individual Blog post story to match. Fixes #84.

### Significant changes

1. Content on Case Studies, Contract Vehicles, and Blog posts now span full width [b503380, 877fb71, 1b22b26].
2. Reduce H1 size on mobile to improve readability.
3. Add spacing to sibling section titles.
4. Rebuild Case Study story from components.

#### H1 Headings

Heading resized for small screens.

| Before | After |
| :----- | :---- |
| image | image |

#### Section title spacing

Now sibling section titles have increased spacing for readability.

| Before | After |
| :----- | :---- |
| image | image|

### CODE REVIEW

File: stories/pages/case-study/case-study.stories.js
Comment (mejiaj) [inline stories/pages/case-study/case-study.stories.js:1]: > [!NOTE]
> This approach gives more freedom to content entry and simplifies the template.

We discussed fixed sections for content sections, so I can also take that approach if that's what we think is best.

For example:
```diff
-    sections: [
-      {
-        variant: "primary-alt",

+   header: {},
+   intro: {},
+   challenge: {},
+   solution: {},
+   impact: {},
+   conclusion: {},
```

**Comment by mejiaj** [comment]
Moving this back to draft to address line length in long form content.

File: stories/pages/case-study/case-study.stories.js
Comment (mattsqd) [reply stories/pages/case-study/case-study.stories.js:1]: @mejiaj  This is totally up to you. If you make it like the individual fields, then you have full control over the placement of the blocks on the page and you can set the header / variant. If you leave it as is, then I'll be the one putting them in the right order and I'll need to set the header / variant. In both ways they'll use the `@components/section/section.html.twig`.

Comment (mejiaj) [reply stories/pages/case-study/case-study.stories.js:1]: @mattsqd let's try the current approach to get the content updated. If we see it's causing issues we can go to a more rigid layout, like the example I put above.

**Review by mattsqd — APPROVED** [review]

**Pull request merged**